docs/admin/kube-apiserver.md

```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
```

Comment:
I don't know why this is getting deleted, but it may be causing the build failure.  Try adding this back in and pushing again.   All these deleted lines.